pkg/cmd/pulumi/stack.go

```diff
@@ -193,28 +195,37 @@ func newStackCmd() *cobra.Command {
 }
 
 func printStackOutputs(outputs map[string]interface{}) {
```

Comment:
Is this even used anymore? Maybe just delete and inline it?

Reply:
Used in one other place. Will delete and inline.